lib/services/compare.go

```diff
@@ -59,9 +59,9 @@ func CompareResources[T any](resA, resB T) int {
 	return Different
 }
 
-// ignoreProtoXXXFields is a cmp.Option that ignores XXX_* fields from proto
+// IgnoreProtoXXXFields is a cmp.Option that ignores XXX_* fields from proto
 // messages.
-func ignoreProtoXXXFields() cmp.Option {
+func IgnoreProtoXXXFields() cmp.Option {
```

Comment:
Should we move this to another package?

Reply:
I'm kinda tempted to pull out the whole comparison/reconciler business out of `services`, but I _so_ do not have the time right now. Not sure where it should go, in any case.